src/System.IO.FileSystem.Watcher/src/System/IO/PatternMatcher.cs

```diff
@@ -140,7 +140,9 @@ public static bool StrictMatchPattern(string expression, string name)
                 // if name is shorter that the stuff to the right of * in expression, we don't
                 // need to do the string compare, otherwise we compare rightlength characters
                 // and the end of both strings.
-                if (name.Length >= rightLength && String.Compare(expression, 1, name, name.Length - rightLength, rightLength, StringComparison.OrdinalIgnoreCase) == 0)
+                if (name.Length >= rightLength && 
+                    String.Compare(expression, 1, name, name.Length - rightLength, rightLength, 
```

Comment:
I'll change these cases.  In general, I'd prefer that cleanup be done with the codeformatter, but a) this library was just posted and these few things obviously slipped through, and b) I'm already editing these files / this project and saw these, and I typically like to operate on a "if you see something, fix it rather than push it off to some pont in the future" mentality, especially when the fixes are trivial.